modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/ShipmentItemResourceImpl.java

```diff
@@ -149,8 +152,16 @@ public ShipmentItem patchShipmentItem(
 			_commerceShipmentItemService.getCommerceShipmentItem(
 				shipmentItemId);
 
+		CommerceInventoryWarehouse commerceInventoryWarehouse =
+			_commerceInventoryWarehouseService.
+				fetchByCommerceInventoryWarehouse(
```

Comment:
probably we have to use the get? because otherwise if this is null (without throwing the error) the `commerceInventoryWarehouse.getExternalReferenceCode()),` will throws NullPointer